Source/Resource/Translation/apd_fr_FR.ts

```diff
@@ -5,284 +5,299 @@
         <source>Click the icon to view battery information, right-click to customize settings or quit.</source>
-        <translation type="unfinished"></translation>
+        <translation>Appuyez sur l&apos;icône pour afficher les informations sur la batterie, appuyez sur le bouton droit pour personnaliser les paramètres ou quitter.</translation>
     </message>
 </context>
 <context>
     <name>DownloadWindow</name>
     <message>
         <source>Download new version</source>
-        <translation type="unfinished"></translation>
+        <translation>Télécharger la nouvelle version</translation>
     </message>
     <message>
         <source>Download Manually</source>
-        <translation type="unfinished"></translation>
+        <translation>Télécharger manuellement</translation>
     </message>
     <message>
         <source>If the download is slow or fails, you can:</source>
-        <translation type="unfinished"></translation>
+        <translation>Si le téléchargement est lent ou échoue, vous pouvez:</translation>
     </message>
 </context>
 <context>
     <name>Gui::DownloadWindow</name>
     <message>
         <source>Oops, an error occurred during the automatic update.
 Please download and install the new version manually.</source>
-        <translation type="unfinished"></translation>
+        <translation>Oups, une erreur s&apos;est produite pendant la mise à jour automatique.
+Nous vous prions de télécharger et d&apos;installer la nouvelle version manuellement.</translation>
     </message>
 </context>
 <context>
     <name>Gui::MainWindow</name>
     <message>
         <source>Change log:</source>
-        <translation type="unfinished"></translation>
+        <translation>Journal des modifications:</translation>
     </message>
     <message>
         <source>Hey! I found a new version available!
 
 Current version: %1
 Latest version: %2%3</source>
-        <translation type="unfinished"></translation>
+        <translation>J&apos;ai trouvé une nouvelle version disponible!
+
+Version actuelle: %1
+Dernière version: %2%3</translation>
     </message>
     <message>
         <source>Bind to AirPods</source>
-        <translation type="unfinished"></translation>
+        <translation>Relier aux AirPods</translation>
     </message>
     <message>
         <source>Please select your AirPods device below.</source>
-        <translation type="unfinished"></translation>
+        <translation>Veuillez-vous sélectionner votre appareil AirPods ci-dessous.</translation>
     </message>
 </context>
 <context>
     <name>Gui::SettingsWindow</name>
     <message>
         <source>|</source>
         <extracomment>To credit translators, you can leave your name here if you wish. (Sorted by time added, separated by &quot;|&quot; character, only single &quot;|&quot; for empty)</extracomment>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Oleh Hnat</translatorcomment>
+        <translation>|</translation>
```

Comment:
Good job! Here the names of the contributors should be filled in the translation instead of the comments.

Reply:
Thanks! You are welcome ^^